requirements.txt

```diff
@@ -24,3 +24,9 @@ scikit-optimize==0.5.2
 
 # Required for plotting data
 #plotly==3.1.1
+
+# find first, C search in arrays
+py_find_1st==1.1.2
+
+#Load ticker files 30% faster
+ujson==1.35
```

Comment:
i'm still not sure if we should "force-depend" on ujson ... 
if we do, we can remove the patchwork-code in backtesting to fall back to the regular json as well.

not subject of this PR though ... just thinking out loud

Reply:
I am not sure that I understand this one :/ ujson is used in Optimize/__init__only and it is renamed to json. what you say is that naming should be explicit so using ujson instead of json when using the library ?

Reply:
no - ujson is currently a optional dependency - benefit from speed if you install it - but works if you don't (and doesn't install by default).
that's why `optimize/__init__.py` does the replacing / renaming.

if we add it to requirements - it's a "mandatory" dependency - failing installation if it can't be installed - in which case we could remove the replacing/renaming in `optimize/__init__.py`.

The reason for this was that it's difficult to install in windows - however wls should work - so we can consider doing that (but it should not be objjective of this PR).